library/games/Senet/0.json

```diff
@@ -0,0 +1,1346 @@
+    "parent": "house7"
+  },
+  "pawn5": {
+    "id": "pawn5",
+    "x": 20,
+    "y": 10,
+    "width": 80,
+    "height": 90,
+    "z": 1,
+    "color": "#749ba5",
+    "css": "border-radius: 40% 40% 50% 50%/ 80% 80% 10% 10%; ",
+    "image": "i/game-pieces/3D/Pawn-3D.svg",
+    "svgReplaces": {
+      "#primaryColor": "color",
+      "#borderColor": "borderColor",
+      "#borderWidth": "borderWidth"
+    },
+    "borderColor": "black",
+    "borderWidth": 1,
+    "resetProperties": {
+      "parent": "house9",
+      "x": 20,
+      "y": 10
+    },
+    "parent": "house9"
+  },
+  "cube2": {
+    "id": "cube2",
+    "x": 20,
+    "y": 10,
+    "width": 80,
+    "z": 1,
+    "color": "#468e9b",
+    "css": "border-radius: 50%/35%;",
+    "image": "/i/game-pieces/3D/Cube-3D.svg",
+    "svgReplaces": {
+      "#primaryColor": "color",
+      "#borderColor": "borderColor",
+      "#borderWidth": "borderWidth"
+    },
+    "borderColor": "#749ba5",
+    "borderWidth": 1,
+    "resetProperties": {
+      "parent": "house4",
+      "x": 20,
+      "y": 10
+    },
+    "parent": "house4"
+  },
+  "cube3": {
+    "id": "cube3",
+    "x": 20,
+    "y": 10,
+    "width": 80,
+    "z": 1,
+    "color": "#468e9b",
+    "css": "border-radius: 50%/35%;",
+    "image": "/i/game-pieces/3D/Cube-3D.svg",
+    "svgReplaces": {
+      "#primaryColor": "color",
+      "#borderColor": "borderColor",
+      "#borderWidth": "borderWidth"
+    },
+    "borderColor": "#749ba5",
+    "borderWidth": 1,
+    "resetProperties": {
+      "parent": "house6",
+      "x": 20,
+      "y": 10
+    },
+    "parent": "house6"
+  },
+  "cube4": {
+    "id": "cube4",
+    "x": 20,
+    "y": 10,
+    "width": 80,
+    "z": 1,
+    "color": "#468e9b",
+    "css": "border-radius: 50%/35%;",
+    "image": "/i/game-pieces/3D/Cube-3D.svg",
+    "svgReplaces": {
+      "#primaryColor": "color",
+      "#borderColor": "borderColor",
+      "#borderWidth": "borderWidth"
+    },
+    "borderColor": "#749ba5",
+    "borderWidth": 1,
+    "resetProperties": {
+      "parent": "house8",
+      "x": 20,
+      "y": 10
+    },
+    "parent": "house8"
+  },
+  "cube5": {
+    "id": "cube5",
+    "width": 80,
+    "z": 1,
+    "color": "#468e9b",
+    "css": "border-radius: 50%/35%;",
+    "image": "/i/game-pieces/3D/Cube-3D.svg",
+    "svgReplaces": {
+      "#primaryColor": "color",
+      "#borderColor": "borderColor",
+      "#borderWidth": "borderWidth"
+    },
+    "borderColor": "#749ba5",
+    "borderWidth": 1,
+    "resetProperties": {
+      "parent": "house10",
+      "x": 20,
+      "y": 10
+    },
+    "x": 20,
+    "y": 10,
+    "parent": "house10"
+  },
+  "stick1": {
+    "id": "stick1",
+    "x": 680,
+    "y": 725,
+    "width": 40,
+    "height": 200,
+    "borderRadius": 15,
+    "z": 1,
+    "css": {
+      "background": "#fff"
+    },
+    "faces": [
+      {
+        "css": {
+          "background": "#444",
+          "border": "3px solid #fff",
+          "rotation": 90,
```

Comment:
I believe this doesn't have any effect. also, the whole face setup could be simplified if you just used a variable for the background color and only changed that in the face

Reply:
The rotation was a leftover accident from testing.  Removed that.  I don't think the css could be simplified enough to make it worth changing, so leaving that as is.